.eslintrc

```diff
@@ -456,6 +457,7 @@
         "before": false,
         "after": true
       }
-    ]
+    ],
+    "@typescript-eslint/lines-between-class-members": ["error"]
```

Comment:
Should we maybe set the option `exceptAfterSingleLine` to `true` (https://github.com/eslint/eslint/blob/main/docs/rules/lines-between-class-members.md#options)? I think it is a bit unnecessary to add an empty line after every variable declaration.

Reply:
Yes, I think it would be better that way. I think I made far too many changes, and I will close this PR and make a new one with the above-stated changes. Would that be good with you?